endpoints.yaml

```diff
@@ -0,0 +1,12 @@
+endpoints:
+  - name: companies
+    path: https://pagsegurohml.img.com.br/server/rest/services/Hosted/Roteirizacao/FeatureServer/2/query
+    requests:
+      - name: list
+        vars:
+          gis: ${{ GIS("https://pagsegurohml.img.com.br/portal", "user", "password") }} 
```

Comment:
We don't have access to external libs inside the python code yet. For now, you can only use the following libs: https://github.com/scanapi/scanapi/blob/master/scanapi/evaluators/code_evaluator.py#L2-L8

Is this value static? If yes, you could store it in a ENV variable: https://scanapi.dev/docs_v1/specification/environment_variables.html

Reply:
This value is not static, it has 60 minutes and then expires and when we need to connect a new one is generated, so this test flow should do the same, hit the endpoint to generate the tokem and the other tasks

Reply:
thanks